lrpar/src/lib/builder.rs

```diff
@@ -125,9 +148,9 @@ pub fn process_file<StorageT, TokId, P, Q>(inp: P,
     outs.push_str(&format!("use lrpar::{{Node, parse_rcvry, ParseError, reconstitute, RecoveryKind}};
 use lrlex::Lexeme;
 
-pub fn parse(lexemes: &Vec<Lexeme<{tn}>>)
-          -> Result<Node<{storaget}, {tn}>, (Option<Node<{storaget}, {tn}>>, Vec<ParseError<{storaget}, {tn}>>)>
-{{", storaget=StorageT::type_name(), tn=TokId::type_name()));
+pub fn parse(lexemes: &Vec<Lexeme<{storaget}>>)
```

Comment:
What does `Lexeme<{storaget}>` mean?

Reply:
It's turned into whatever type the user specified. So if you specified `u8` this would generate code `pub fn parse(lexemes: &Vec<Lexeme<u8>>)`.